core/src/test/scala/integration/kafka/api/AdminClientTest.scala

```diff
@@ -97,6 +113,15 @@ class AdminClientTest extends IntegrationTestHarness with Logging {
   }
 
   @Test
+  def testDescribeGroupTimeout() {
+    try {
+      clientTimeout.describeGroup(groupId);
+    } catch {
+      case _: TimeoutException =>
```

Comment:
This won't fail if no exception is thrown, you should probably pass the expected exception to the @Test annotation and remove the catch (the alternative is to add a `fail` after the `describeGroup` call, but it's more verbose. Other tests seem to have a similar issue.